### DISCUSSION TITLE
Instagram Issue. Pinwheel.

### DISCUSSION THREAD

**Discussion by itsknowone** [qsniyg/maxurl#1032] [Q&A] [answered]
I used to be able to run this extension on Instagram images but now it just pinwheels. I am on PC, running this through Firefox. Not sure why its breaking.

**Comment by qsniyg** [comment, marked as answer]
To keep things simple, I've replied at #1023, I believe it's the same issue you're experiencing.

Edit: It should now be completely fixed :)
